src/govuk/components/exit-this-page/_index.scss

```diff
@@ -56,13 +56,40 @@
     }
   }
 
+  .govuk-exit-this-page__indicator-light {
+    display: inline-block;
+    position: relative;
+    width: $indicator-size;
+    height: $indicator-size;
+    margin: 0 .125em;
+    border-width: $indicator-border-width;
+  }
+
+  // High contrast/forced colour modes tend to remove background colours, so we
+  // create the 'lit' indicator  entirely out of 'border' so that they remain
+  // 'filled' even when colours are being overridden.
+  .govuk-exit-this-page__indicator-light:after {
```

Comment:
I think you could potentially avoid the after pseudo element by changing the size and border width of the indicator element instead?

Something like:

```scss
  .govuk-exit-this-page__indicator-light {
    display: inline-block;
    position: relative;
    width: $indicator-size;
    height: $indicator-size;
    margin: 0 .125em;
    border-width: $indicator-border-width;
  }

  .govuk-exit-this-page__indicator-light--on {
    width: 0;
    height: 0;
    border-width: ($indicator-size / 2) + $indicator-border-width;
  }
```

Reply:
Happy to go that route instead! Would negate the need for an IE8-specific fix.

Could probably negate the need for fancy mathematics by using `box-sizing: border-box` instead too.